backend/handlers/handlers.go

```diff
@@ -226,6 +229,8 @@ func NewHandlers(
 	getAPIRouterNoError(apiRouter)("/exchange/pocket/api-url", handlers.getExchangePocketURL).Methods("GET")
 	getAPIRouterNoError(apiRouter)("/exchange/pocket/sign-address", handlers.postPocketWidgetSignAddress).Methods("POST")
 	getAPIRouterNoError(apiRouter)("/exchange/pocket/verify-address", handlers.postPocketWidgetVerifyAddress).Methods("POST")
+	getAPIRouterNoError(apiRouter)("/bitsurance/lookup", handlers.getBitsuranceLookup).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/bitsurance/url/{code}", handlers.getBitsuranceURL).Methods("GET")
```

Comment:
why is it part of the URL here and a GET param in the lookup endpoint? should ideally be consistent